src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -274,7 +274,7 @@ public async Task<IActionResult> Create(EditTenantViewModel model)
 
                 shellSettings.Name = model.Name;
                 shellSettings.RequestUrlHost = model.RequestUrlHost;
-                shellSettings.RequestUrlPrefix = model.RequestUrlPrefix?.Trim();
+                shellSettings.RequestUrlPrefix = model.RequestUrlPrefix?.Trim(' ', '/');
```

Comment:
im wondering - because we do this in 3 places, it might be worth creating a method on the model, or doing this within the getter on the model

Reply:
Okay, i will work on it this night.

Note: i forgot to remove the one in `ModularBackgroundService`, i wanted to keep it only in 2 places (as before with only `.Trim()`), only when we update it from a model (2 different models).

Hmm, finally maybe better to do this when we get it from the shell settings because now the value might come from configuration, a preconfigured value which might have spaces or `/`.